tests/test_form.py

```diff
@@ -25,6 +25,16 @@ def test_form(testcase, preconstruct):
             form.data = doc
 
 
+@pytest.mark.parametrize("preconstruct", (0, 1))
```

Comment:
No need to test this with this parametrization, as the two features do not interact.